packages/letters/letters.0.1.0/opam

```diff
@@ -0,0 +1,42 @@
+opam-version: "2.0"
+synopsis: "Client library for sending email over SMTP"
+description: "Simple to use SMTP client implementation for ocaml"
+maintainer: ["Miko Nieminen <miko.nieminen@iki.fi>"]
+authors: ["Miko Nieminen <miko.nieminen@iki.fi>"]
+homepage: "https://github.com/oxidizing/letters/"
+bug-reports: "https://github.com/oxidizing/letters/issues"
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {>= "2.0"}
+  "mrmime" {= "0.3.0"}
+  "colombe" {= "0.3.0"}
```

Comment:
Sorry, one last question. Can we relax these three bounds with a `>=`? You mentioned that there are incoming fixes for example to colombine which you will benefit from/

Reply:
Ignore my comment, I have just seen your response in the other thread

Reply:
The API of `colombe` and I think also `mrmime` changes with every release as it is very early release and I had some head scratching why for example `facteur` (example app using `colombe`) didn't work until I realized that it needed old version of `colombe`. That's why I fixed these version now, but I suppose I could say something like `{ >= "0.3.0" & < "0.4.0" }`.

Reply:
`{ >= "0.3.0" & < "0.4.0" }` would have been a bit better.
In any case, please note that strict requirements are just going to annoy your users and you in the long run. New `mrmime`/`colombe`/... packages are checked in our CI for breakage, and we or the package maintainers (in this case @dinosaure) do the job of updating the constraints if needed.